.github/workflows/e2e-tests.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           build: npm run build --workspaces --if-present
           start: npm run start-test-env
-          wait-on: http://localhost:8888
+          wait-on: 'http://localhost:8888/wp-admin/index.php, http://localhost:8889'
```

Comment:
Can you explain why we would be waiting on he `admin.php/index.php` file? Doesn't that trigger a redirect to the `wp-login.php` URL?

Reply:
This line _should_ be helping fix some of the tests happening before the WP environment is ready. In the Cypress screenshots, it's been using the environment before WP fully installs. I don't think we'd strictly _have_ to use this URL, but we specifically need to make sure it can get to the actual admin and doesn't end up in `install.php`.

Reply:
Okay cool :) Yeah as long as it is working this is all great :) Was just curious how this change helped fix it :)